src/libraries/System.Text.Json/src/readme.md

```diff
@@ -0,0 +1,7 @@
+## About
```

Comment:
IMHO I find this document underwhelming without usage examples in it. Also, "Commonly Used Types" are part of the package description but not of the README.md file. Is there a reason for that?

Reply:
Examples were included initially but I asked they be removed in favor of using docs as our source of truth.

Reply:
> Examples were included initially but I asked they be removed in favor of using docs as our source of truth.

The Commonly Used Types were already part of the package description before so I think we should have them.

> I didn't touched package description currently. Would it make sense to update it to be the same as first two paragraphs of readme (without links)?

Yes. I would move the Commonly Used Types into the README.md file and update the description to what you are proposing.

Reply:
The list of commonly used types in `PackageDescription` is out of date, for example it does not include `JsonSerializerContext` which is essential for source generation since .NET 6. This goes back to my point of such lists eventually going out of sync. In any case, I don't see a lot of value in simply enumerating the common types: you can achieve much more with proper, curated documentation or just intellisense.

Reply:
That's a fair point and makes sense to me. If so, then we should update the package description to remove them from there.

Reply:
Updated.